spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -12,17 +12,15 @@ def create_generic_work
   end
 
   def create_atlas_work
-    generate 'hyrax:work RareBooks/Atlas'
     # ActiveSupport interprets "atlas" as plural which causes
     # counter-intuitive route paths. Add an inflection to correct
     # these paths
     append_file 'config/initializers/inflections.rb' do
-      <<~HEREDOC
-        ActiveSupport::Inflector.inflections(:en) do |inflect|
-          inflect.irregular 'atlas', 'atlases'
-        end
-      HEREDOC
+      "ActiveSupport::Inflector.inflections(:en) do |inflect|\n" \
+      "  inflect.irregular 'atlas', 'atlases'\n" \
+      "end\n"
     end
+    generate 'hyrax:work RareBooks/Atlas'
```

Comment:
Also, setting up the inflections *after* generating the model was kinda silly. Caused test failures.